### PR TITLE
Set config directory and init logging config

### PR DESCRIPTION
**Pull request by sswguo**
We need to generate the corresponding ConfigMaps from the existing ispn config files for auditquery, 
and the ConfigMaps will be mounted as files on the pod, and we can load them as if they were just fully-resolved configuration files.

### CODE REVIEW

File: deployments/standalone-rest/src/main/scripts/start.sh [hunk 1]
```diff
@@ -20,6 +20,12 @@ THIS=$(cd ${0%/*} && echo $PWD/${0##*/})
 BASEDIR=`dirname ${THIS}`
 BASEDIR=`dirname ${BASEDIR}`
 
+AQ_LOGCONF_DIR=${INDY_LOGCONF_DIR:-${BASEDIR}/etc/auditquery/logging}
```
Comment (jdcasey) [inline deployments/standalone-rest/src/main/scripts/start.sh]: Let's change the logging directory to a generic CLASSPATH_DIR so we can put other things in there if necessary. We had to control the Indy Infinispan JDBC c3p0 pools using a file in the logging config dir once, for example...that was awkward. Also, there's an Indy reference here... :grin: 

Hmm...now that I think about it, we could put custom infinispan.xml files here too...maybe we want to just make this the conf directory, and add it to the classpath?

File: deployments/standalone-rest/src/main/scripts/start.sh [hunk 2]
```diff
@@ -20,6 +20,12 @@ THIS=$(cd ${0%/*} && echo $PWD/${0##*/})
...
+
+echo "Loading logging config from: ${AQ_LOGCONF_DIR}"
```
Comment (jdcasey) [inline deployments/standalone-rest/src/main/scripts/start.sh]: Maybe "Loading extra config from classpath directory: ..." instead?

File: deployments/standalone-rest/src/main/conf/main.conf [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # API version
 
+config.dir=${auditquery.home}/etc/auditquery/
```
[the remark this answers is not in the document]
Comment (sswguo) [reply deployments/standalone-rest/src/main/conf/main.conf]: Yeah, agree, at beginning i just want to make this consistent with indy way. But sure, it's better to make it simple.

File: deployments/standalone-rest/src/main/scripts/start.sh (hunk 1 above)
Comment (sswguo) [reply deployments/standalone-rest/src/main/scripts/start.sh]: Ah, bad to see indy reference. 😄 
Currently i just add the conf directory to the classpath.

File: deployments/standalone-rest/src/main/scripts/start.sh (hunk 2 above)
Comment (sswguo) [reply deployments/standalone-rest/src/main/scripts/start.sh]: Sure. 👍

**Pull request merged**